lib/views/help/unhappy.html.erb

```diff
@@ -46,34 +46,60 @@
-  authority, then  <a href="http://www.itspublicknowledge.info/YourRights/YourRights.asp">appeal
-  to the Scottish Information Commissioner</a>.
+<a name="complaining"></a>
+<h2 id="refer-to-ico">2. Referring a request to the Information Commissioner <a class="hover_a" href="#refer-to-ico">#</a></h2>
+<p>
+  If you are still unhappy after the public authority has done their internal
+  review, then you can refer your request to the Information Commissioner, who
+  is empowered to uphold access to information laws. To make a referral,
+  start by reading <a href="https://ico.org.uk/concerns/getting/">the
+  Information Commissioner's advice for those with concerns about accessing
+  information</a> which links to their form. If you requested information from a
+  Scottish authority, then it is
+  <a href="http://www.itspublicknowledge.info/YourRights/YourRights.asp">the
+  Scottish Information Commissioner</a> who you will need to contact.
 </p>
-<p>To make it easier to send the relevant information to the
-  Information Commissioner, either
-  <% if !@info_request.nil? %>
-  include a link to your request
-  <strong><%= request_url(@info_request) %></strong>
-  <% else %>
-  include a link to your request on WhatDoTheyKnow
-  <% end %>
-  in your complaint or print out the whole page of your request and all attachments.
+
+<p>
+  While the Information Commissioner asks for their form to be completed and
+  evidence supporting a referral to be attached in practice we are aware they
+  routinely consider referrals made via a simple email setting out the issues,
+  containing a link to the request on WhatDoTheyKnow.com to provide evidence of
+  the full history of the correspondence related to the request. If you do wish
+  to download copies of correspondence, and attachments, to send to the
+  Information Commissioner the <code>.zip</code> download feature, accessed via
+  "Actions" may assist.
+</p>
+
+<p>
+  WhatDoTheyKnow does not currently have any special facilities for handling a
+  referral to the Information Commissioner. We encourage users to leave
+  annotations on requests keeping people informed of progress.
+  <a href="https://icosearch.ico.org.uk/s/search.html?collection=ico-meta&profile=decisions&query">
+  Decisions by the Information Commissioner are published online</a> and you can
+  link to a decision via an annotation on the request.
+</p>
+
+<p>
+  A warning. Although the Information Commissioner has worked hard to reduce
+  their backlog of casework, it can still take several months to get resolution
+  from them in many cases. One area where they have sped up things considerably
+  is that they are able to prompt non-responsive authorities to reply within a
+  few weeks of receiving a complaint.
+</p>
+
+<p>
+  If you are unhappy with the response from the Information Commissioner's
+  office you can ask them to reconsider it under their complaints process.
```

Comment:
Richard's text suggested linking "complaints process" to `
https://ico.org.uk/concerns/complaints-and-compliments-about-us/complain-about-us/`

Reply:
😳 oops, nope, missed that